### DISCUSSION TITLE
Time Series Anomaly Detection

### DISCUSSION THREAD

**Discussion by aryanxk02** [open-edge-platform/anomalib#1119] [Feature Requests]
Hi, I would like to know if there is a `Time Series Anomaly Detection` model present in Anomalib codebase. If yes, please share me the link else if there are any plans to build a model – 
[1] What is the dataset that is being considered for developing such model (LSTM/GRU) ?
[2] Will this model be trained on multiple features i.e Multivariate LSTM ?

Thanks and Regards

**Comment by samet-akcay** [comment]
Hi @aryanxk02, I think the following paper would be of interest since it proposes a model that could be used in image and time series domains, respectively.

https://paperswithcode.com/paper/set-features-for-fine-grained-anomaly

**Comment by aryanxk02** [comment]
Thanks for sharing, I will go through it @samet-akcay
